.circleci/config.yml

```diff
@@ -25,12 +25,12 @@ jobs:
 
 
       - run:
-          name: Moving the bears app into usagov-2021 module
+          name: Moving the benefit-finder app into usagov-2021 module
           command: |
             cd px-bears-drupal
-            bash ./mv-bears-app.sh
+            bash ./mv-benefit-finder-app.sh
             echo ".................."
-            echo "moving the bears-app module into the usagov-2021 directory"
+            echo "moving the benefit-finder-app module into the usagov-2021 directory"
             mv  usagov_bears usagov-2021/web/modules/custom
```

Comment:
@scottqueen-bixal Do you think I need to change the command here? Like, the directory name?
mv usagov_bears usagov-2021/web/modules/custom

Reply:
`usagov_bears` will change, @gchi25 what are you going to use here for directory? `usagov-benefit-finder` ?

Which ever we end up with in the module update, we will need to update here in this script as well, so be sure to coordinate.